### PR TITLE
Vctmsvcs 37 list of providers page

### PR DESCRIPTION
**Pull request by bdwoodruff**
This branch includes the List of providers task and should have everything plumbed according to the flow Kristen has set up as of right now.  the page itself is a tabbed page with a map and annotation in one tab and a list of providers in the other tab.

### CODE REVIEW

**Comment by bdwoodruff** [comment]
This seems to be adding files that I thought were already committed and merged. Not sure why, but I guess we'll get it figured out.

File: app/styles/serviceProviders.tss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	font:{
+		fontSize: '24dp',
+		fontWeight: 'bold'
+	}
+}
+
+"#headerLeft":{
+	left: "6%",
+	width: "11%"
+	//left: "30dp",
```
Comment (bachmakm) [inline app/styles/serviceProviders.tss]: Remove commented styles. Same goes for comments below.

File: app/styles/serviceProviders.tss [hunk 2]
```diff
@@ -0,0 +1,83 @@
+".container" : {
+  
+}
+
+"#header": {
+	backgroundColor: "#00a1ff",
+	opacity: 1,
+	color: "#0e4bea",
+	height: "8%",
+	//height: "50dp",
```
Comment (bachmakm) [inline app/styles/serviceProviders.tss]: plz remove me and my other commenty friends!

EDIT: oops... thought I was commenting a different file.

**Comment by bachmakm** [comment]
Overall looks good, but it won't run on my machine. :(

**Comment by bdwoodruff** [comment]
I think it might be because the service providers was folded into the listOfProviders page on the list tab.

**Pull request merged**